lib/evmone/advanced_instructions.cpp

```diff
@@ -8,8 +8,16 @@
 
 namespace evmone::advanced
 {
+/// Fake wrap for generic instruction implementations accessing current code location.
+/// This is to make any op<...> compile, but pointers must be replaced with Advanced-specific
+/// implementation. Definition not provided.
+template <code_iterator InstrFn(ExecutionState&, code_iterator)>
+const Instruction* op(const Instruction* /*instr*/, AdvancedExecutionState& state) noexcept;
```

Comment:
Can this be `= 0` maybe?

Reply:
Do you mean `= delete`? I don't think so. The trick here is that this is actually used during compilation but later overwritten by Advanced's variants so linker does not complain. So this has to be external function definition (I even needed to move it out of anonymous namespace).